src/view/shell/index.tsx

```diff
@@ -144,7 +146,7 @@ export const Shell: React.FC = function ShellImpl() {
     }
   }, [theme])
   return (
-    <View testID="mobileShellView" style={[styles.outerContainer]}>
+    <View testID="mobileShellView" style={[styles.outerContainer, pal.view]}>
```

Comment:
FYI you can use `useTheme` from ALF and `t.atoms.bg` for this too 👍

Reply:
ah well, next time :P